src/components/donation/Steps/Submit/StripeCheckout/StripeCheckout.tsx

```diff
@@ -40,6 +41,15 @@ export default function StripeCheckout(props: StripeCheckoutStep) {
       amount={+details.amount}
       splitLiq={liquidSplitPct}
     >
+      <div className="mb-2">
```

Comment:
```suggestion
      <div className="mb-6">
```
nit: `mt-4` can be removed from the neighboring `div`.

Reply:
applied nit